site/es/blog/new-in-devtools-92/index.md

```diff
@@ -0,0 +1,224 @@
+
+Chromium issue: [1201398](https://crbug.com/1201398)
+
+### Filtra los recursos de tipo Wasm en el panel de *Network* {: #wasm }
+
+Ahora puedes filtrar los recursos de red de Web Assembly usando el botón *Wasm* en el panel de *Network*.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/vuTMcfCjDWFfVtDN6Dpf.png", alt="Filtro para Wasm", width="800", height="515" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/db3b40161aeb0856d33e0e4545b2b5bd8e79fb86 #}
+
+Chromium issue: [1103638](https://crbug.com/1103638)
+
+### User-Agent Client Hints por dispositivo en la pestaña de condiciones  {: #sec-ua-ch }
+
+Ahora los [User-Agent Client Hints](https://web.dev/user-agent-client-hints) son aplicados por dispositivo en el campo User Agent debajo de la pestaña de **Network Conditions**.
+
+**Los User-Agent Client Hints son una nueva expansión de la Client Hints API.** Esto permite a los desarrolladores a acceder a información sobre el navegador del usuario y su dispositivo preservando la privacidad del usuario y de una forma mucho más ergonómica.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/iMlkTtV9OUdfujSWdHnR.png", alt="User-Agent Client Hints para dispositivos en la pestaña Network", width="800", height="532" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/b2b0c3b6c6e093649c35b6824004284ca4c2bd4a #}
+
+Chromium issue: [1174299](https://crbug.com/1174299)
+
+## Reporta problemas con el modo Quirks en la pestaña de Issues {: #quirks-mode }
+
+DevTools ahora reporta los posibles problemas con el [modo Quirks](https://quirks.spec.whatwg.org/) y el [modo Limited-Quirks](https://dom.spec.whatwg.org/#concept-document-limited-quirks).
+
+El modo Quirks y Limited-Quirks son modos de navegadores antiguos cuando todavía los estándares web no fueron establecidos. Estos modos emulan la era pre-standard para como se comportaba el diseño en el navegador y en ocasiones causan efectos visuales inesperados.
+
+Cuando depuras problemas de diseño, los desarrolladores pueden pensar que son causados por CSS que han producido o errores en el HTML, mientras que el problema en realidad está en el modo de compatibilidad que la página se encuentra. Ahora DevTools proporciona sugerencias para solucionarlo.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/XqtqSZPa1S1YnmeIt0ee.png", alt="Reporta los problemas del modo Quirks en la pestaña Issues", width="800", height="490" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/79d5f4274f21bb78e2ce572d118c2bd5bf1cfa82 #}
+
+Chromium issue: [622660](https://crbug.com/622660)
+
+## Incluye las intersecciones computadas en el panel de Performance {: #computed-intersections }
+
+DevTools ahora te muestra el coste de las **intersecciones computadas** en la tabla de llamas. Este cambio te puede ayudar a identificar los eventos de [Intersection Observer](https://web.dev/intersectionobserver-v2/) y depurar cualquier problema potencial de rendimiento.
```

Comment:
I think "tabla de llamas" is not the proper translation for "flame chart". Since a translation for the chart's name hasn't been widely adopted yet , I suggest we refer to it in another manner:
```suggestion
DevTools ahora te muestra el coste de las **intersecciones computadas** en la pila de JavaScript. Este cambio te puede ayudar a identificar los eventos de [Intersection Observer](https://web.dev/intersectionobserver-v2/) y depurar cualquier problema potencial de rendimiento.
```